personalsite/src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import { MenuItems } from "./MenuItems"
+import { Button } from "../Button"
+import './Navbar.css'
+
+class Navbar extends Components {
```

Comment:
`class Navbar extends Component` not `Components` --> this is why it was breaking